projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -484,8 +484,11 @@ public function add_gutenberg_menus() {
 
 	/**
 	 * Add the calypso /woocommerce-installation/ menu item.
+	 *
+	 * @param array $current_product The site's product if they have one. This is passed from WPcom_Admin_Menu to
+	 * prevent redundant database queries.
 	 */
-	public function add_woocommerce_installation_menu() {
+	public function add_woocommerce_installation_menu( $current_product = null ) {
```

Comment:
We don't need the current plan information on the Atomic side?

Reply:
No. Not currently at least, because we're showing the WooCommerce installation menu item to all Atomic sites, regardless of plan, with ["activate_plugins" enabled](https://github.com/Automattic/jetpack/blob/8f7b78eeab9955eacb6a36b6d4787269fda19125/projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php#L507). Consequently, this turns out to be Business plans.